ui/raidboss/data/02-arr/trial/titan-nm.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.TheNavel,
+  timelineFile: 'titan-nm.txt',
+  triggers: [
+    {
+      id: 'TitanNm Tumult',
+      netRegex: NetRegexes.ability({ id: '282', source: 'Titan', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      suppressSeconds: 2,
+      response: Responses.aoe(),
+    },
+    {
+      // Gaol callout for both yourself and others
+      id: 'TitanNm Gaols',
+      netRegex: NetRegexes.gainsEffect({ effectId: '124' }),
+      alertText: function(data, matches) {
+        if (matches.target !== data.me) {
+          return {
+            en: 'Break Gaol on ' + data.ShortName(matches.target),
+          };
+        }
+        return {
+          en: 'Gaol on YOU',
+        };
+      },
+    },
+    {
+      // Covers both tanks and non tanks with appropriate text
+      // Needs suppression for occasions in phase 4 where rock busters
+      // get pushed close together by auto attacks to avoid double triggers.
+      id: 'TitanNm Rock Buster',
+      netRegex: NetRegexes.ability({ id: '281', source: 'Titan' }),
+      condition: (data) => data.role !== 'tank' || data.role === 'tank',

```

Comment:
I think this is a tautology, and maybe you can remove the entire `condition` line?

Reply:
Will removing the condition allow it to capture the roles still and return the correct tank cleave alert? I wrote it that way just to be sure it captures player roles correctly to assign them the correct alert based on their role.

Reply:
This way it applies Tank Cleave on YOU to the tank, and Avoid Tank Cleave to everyone else, and it does it in 1 trigger instead of two

Reply:
`TankCleave() already handles that logic internally. Because this will be a timeline trigger, we won't have access to the `target`, so the `response` will just look at roles without needing explicit input from the trigger.

Reply:
Oh cool, ty!